routes/recordOverview.js

```diff
@@ -3,34 +3,101 @@ var linz = require('linz'),
     async = require('async'),
     clone = require('clone');
 
+
 /* GET /admin/:model/:id/overview */
 var route = function (req, res, next) {
 
     var locals = {
         model: req.linz.model,
         record: clone(req.linz.record.toObject({ virtuals: true})),
         permissions: req.linz.model.linz.formtools.permissions,
-        formtools: req.linz.model.linz.formtools
+        formtools: req.linz.model.linz.formtools,
+        tabs: []
     };
 
     async.series([
 
         function (cb) {
 
-            req.linz.model.linz.formtools.overview.summary.renderer(req.linz.record, req.linz.model, function (err, content) {
+            var details = req.linz.model.linz.formtools.overview.details;
+
+            if (typeof details === 'function') {
+
+                return req.linz.model.linz.formtools.overview.details(req, res, req.linz.record, req.linz.model, function (err, content) {
+
+                    if (err) {
+                        return cb(err);
+                    }
+
+                    locals.customOverview = content;
+                    return cb();
+                });
+            }
+
+            linz.formtools.overview.getOverviewFields(req.linz.model.schema, req.linz.model.linz.formtools.form, details, req.linz.record, req.linz.model, function (err, fields) {
 
                 if (err) {
                     return cb(err);
                 }
 
-                locals.overviewSummary = content;
-
-                return cb(null);
+                locals.fields = fields;
 
+                return cb();
             });
 
         },
 
+        // process overview tabs
+        function (cb) {
+
+            if (!Array.isArray(req.linz.model.linz.formtools.overview.tabs) || !req.linz.model.linz.formtools.overview.tabs.length) {
+
+                return cb();
+            }
+
+            async.eachSeries(req.linz.model.linz.formtools.overview.tabs, function (tab, callback) {
+
+                if (typeof tab.body === 'function' ) {
+
+                    return tab.body(req, res, req.linz.record, req.linz.model, function (err, content) {
+
+                        if (err) {
+                            return callback(err);
+                        }
+
+                        locals.tabs.push({
+                            label: tab.label,
+                            body: content
+                        });
+
+                        return callback();
+                    });
+
+                }
+
+                if ( !Array.isArray(tab.fields) || !tab.fields.length ) {
```

Comment:
Please remove the preceding and post spaces within the brackets (i.e. `if (!Array.isArray(tab.fields) || !tab.fields.length) {`).